statsmodels/iolib/smpickle.py

```diff
@@ -19,7 +19,12 @@ def save_pickle(obj, fname):
 
 def load_pickle(fname):
     """
-    Load a previously saved object from file
```

Comment:
Could you use the nonbold here too, as in the other two examples? Or is there some reason for bold here?

Reply:
The bold is to emphasize the warning: this one is the most important of the 3 because it's the only one that is listed in the documentation.